statsmodels/genmod/generalized_linear_model.py

```diff
@@ -963,9 +985,9 @@ def _fit_irls(self, start_params=None, maxiter=100, tol=1e-8,
 
         # first guess on the deviance is assumed to be scaled by 1.
         # params are none to start, so they line up with the deviance
-        history = dict(params=[None, start_params], deviance=[np.inf, dev])
+        history = dict(params=[np.inf, start_params], deviance=[np.inf, dev])
```

Comment:
Wait now that I think about it, I think it was actually `allclose` that was in one of my tests. See this line. 

https://github.com/statsmodels/statsmodels/pull/2803/files#diff-bacf4e7dec296273aba74b04be3c42f8R1687

I presume I could just have it skip the first test... Duh. Let me fix that.